tidy3d/web/api/container.py

```diff
@@ -296,7 +397,32 @@ def load(cls, path_dir: str = DEFAULT_DATA_DIR) -> BatchData:
 
 
 class Batch(WebContainer):
-    """Interface for submitting multiple simulations to the sever."""
+    """
+    Interface for submitting several :class:`Simulation` objects to sever.
+
+    Notes
+    -----
+
+        Commonly one needs to submit a batch of :class:`Simulation`. The built-in :class:`Batch` object is the best way to upload,
+        start, monitor, and load a series of tasks. The batch object is like a :class:`Job`, but stores task metadata
+        for a series of simulations.
+
+    See Also
+    --------
+
+    :meth:`tidy3d.web.run_async`
+        Submits a set of :class:`.Simulation` objects to server, starts running, monitors progress,
+        downloads, and loads results as a :class:`.BatchData` object.
+
+    :class:`Job`:
+        Interface for managing the running of a Simulation on server.
+
+    **Notebooks**
+        * `Running simulations through the cloud <../../notebooks/WebAPI.html>`_
+        * `Performing parallel / batch processing of simulations <../../notebooks/ParameterScan.html>`_
+        * `Parameter Scan using signac <../../notebooks/ParameterScan.html#Parameter-Scan-using-Signac>`_
```

Comment:
probably dont want to focus on the signac part of this, just the regular notebook does a `Batch` (I dont think the signac part does).